corehq/apps/userreports/document_stores.py

```diff
@@ -6,12 +6,12 @@
 from pillowtop.dao.couch import CouchDocumentStore
 
 
-def get_document_store(domain, doc_type):
+def get_document_store(domain, doc_type, case_type_or_xmlns=None):
     use_sql = should_use_sql_backend(domain)
     if use_sql and doc_type == 'XFormInstance':
-        return ReadonlyFormDocumentStore(domain)
-    elif use_sql and doc_type == 'CommCareCase':
-        return ReadonlyCaseDocumentStore(domain)
+        return ReadonlyFormDocumentStore(domain, xmlns=case_type_or_xmlns)
+    elif doc_type == 'CommCareCase':
```

Comment:
At the moment the SQL based document stores do not support last_id. last_id is used to resume builds, but I think filtering will be much more useful for couch based document stores so removing the check from here